### PR TITLE
Fix one-liner envvars for Windows powershell/cmd. Resolve #341.

### LINKED ISSUE #341: Environment variables in PowerShell
https://community.k6.io/t/changing-new-relic-statsd-metrics-prefix-k6/1417/6 made it apparent that the way to set environment variables in PowerShell is different:
- https://stackoverflow.com/questions/1420719/powershell-setting-an-environment-variable-for-a-single-command-only
- https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.1

We should probably add a note about that in https://k6.io/docs/using-k6/environment-variables/ and https://k6.io/docs/using-k6/options/

### PR DESCRIPTION
**Pull request by Griatch**
This aims to resolve https://github.com/grafana/k6-docs/issues/341. That issue indicates problems with the Powershell one-liners, but looking at the code blocks suggested issues with the CMD blocks as well - notably they tried to specify the envvars on two lines using `\n` which was ignored in the verbatim field (all ended up as one line and looked very messy).  

To set up one-liner Windows CMD blocks, I referenced this: 
https://stackoverflow.com/questions/23420340/windows-command-prompt-using-a-variable-set-in-the-same-line-of-a-one-liner

I also updated the Powershell examples to use one-liners, using the links suggested in the issue:
https://stackoverflow.com/questions/1420719/powershell-setting-an-environment-variable-for-a-single-command-only

I however have no Windows machine easily available to test these examples. :/

### CODE REVIEW

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (Griatch) [inline src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:522]: Not strictly part of the PR, but it seemed unnecessary to have a line number for this.

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (Griatch) [inline src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:972]: This had no system separation at all before.

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (MStoykov) [inline src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:1015]: I am not really of the opinion that we should have an example of doing this at all ... or have the functionality as well so ... slight :-1: and request to just remove the documentation around `K6_STAGES`.  Possibly not in this PR

**Review by MStoykov — APPROVED** [review]
I also haven't tested this on windows, but if it says it should be something like that it's probably correct ... and it's definitely better than what we currently have :rofl:

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (Griatch) [reply src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:1015]: I imagine if it becomes a new PR it'll take equally long time before it's actually removed :D It's easy enough to remove. I'll leave it up to master @ppcano to decide.

File: src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md
Comment (Griatch) [inline src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md:42]: The `\n` is ignored here since it's inside a verbatim field.

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (ppcano) [reply src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:1015]: @MStoykov Please, can you open a new issue?

File: src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md
Comment (tom-miseur) [inline src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md:46]: Slightly cleaner without the spaces:
```
PS C:\k6> $env:MY_HOSTNAME="test.k6.io"; k6 run script.js
```

**Review by tom-miseur — COMMENTED** [review]
I've verified the commands work as expected for both CMD and PowerShell. I've noted a few minor syntax issues.

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (Griatch) [reply src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:150]: Great - I'll update the prompts used. :+1:

File: src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md
Comment (Griatch) [reply src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md:42]: You are commenting on a removed line though - I think I removed the escaped quotes already?

File: src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md
Comment (Griatch) [reply src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md:46]: Will do, thanks!

File: src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md
Comment (tom-miseur) [reply src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md:42]: Oops!

**Comment by Griatch** [comment]
Updated after feedback from @tom-miseur. Thanks!

**Comment by ppcano** [comment]
@tom-miseur, approved PR?

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (tom-miseur) [inline src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:156]: Sorry -  missed this one earlier:

```suggestion
PS C:\k6> $env:K6_NO_CONNECTION_REUSE="true"; $env:K6_USER_AGENT="MyK6UserAgentString/1.0"; k6 run script.js
```

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (tom-miseur) [inline src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:1003]:
```suggestion
C:\k6> set "K6_STAGES=5s:10,5m:20,10s:5" && k6 run script.js
```

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (tom-miseur) [inline src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:1008]:
```suggestion
C:\k6> k6 run --stage 5s:10,5m:20,10s:5 script.js
```

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (tom-miseur) [inline src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:1012]:
```suggestion
C:\k6> $env:K6_STAGES="5s:10,5m:20,10s:5"; k6 run script.js
```

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (tom-miseur) [inline src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:999]: The tilde isn't a thing in CMD or PowerShell:

```suggestion
C:\k6> k6 run --stage 5s:10,5m:20,10s:5 script.js
```

**Review by tom-miseur — CHANGES_REQUESTED** [review]
Some more stuff I missed last round. Sorry!

**Comment by Griatch** [comment]
@tom-miseur Add the fixes. Also realized one block on that options page didn't have any windows cmdline examples at all, so I added that (this is beyond the PR overall, really, but it makes sense to add).

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (Griatch) [reply src/data/markdown/translated-guides/en/02 Using k6/05 Options.md]: Nice catch - that's me having clumsy fingers.

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md
Comment (Griatch) [reply src/data/markdown/translated-guides/en/02 Using k6/05 Options.md]: This is found all over the place; not sure why it was added in the first place when it seems we are expected to be in a special `k6` folder. Removed.

**Review by tom-miseur — APPROVED** [review]
LGTM!

**Comment by ppcano** [comment]
Thanks, everyone!

**Pull request merged**
